.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] "
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. step 1
+2. step 2
+3. step 3
+4. etc.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```

Comment:
What does "Desktop" mean?

Reply:
@sorawee it asks the submitter to provide the OS& version and the racket version.


--
Desktop (please complete the following information):

OS: [Please include version, etc. e.g. Windows 7, 32 bit, MacOS Catalina 10.15.2, Debian x86-64, Raspbian etc.]
Racket Version [e.g. 7.5]
Additional context Add any other context about the problem here.

Reply:
I understand the below two lines (OS and Racket version), but I don't understand what "Desktop" means. If I use a laptop computer, not a desktop computer, then I don't need to fill in this section?

In any case, this is off-topic. Do you agree that this PR should be closed, and #3130 should be used instead?

Reply:
yes